appengine_flexible/datastore/datastore.go

```diff
@@ -42,6 +42,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	defer client.Close()
```

Comment:
Looks like this should be datastoreClient. Might be other cases like this.

Reply:
Done.